.github/workflows/scheduled.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test-strategy: ["min", "release", "max"]
+        test-strategy: ["min_cohort", "min_isolated", "release", "max"]
```

Comment:
When running `min_isolated_deps_check_result <- verdepcheck::min_isolated_deps_check("teal.widgets")` it gives this error:

```r
✔ Installed Rcpp 1.0.11  (309ms)
✔ Installed R.utils 2.12.2  (105ms)
✔ Installed R.oo 1.25.0  (106ms)
✔ Installed R.methodsS3 1.8.2  (106ms)
✔ Installed R.cache 0.16.0  (106ms)
✔ Installed AsioHeaders 1.22.1-2  (161ms)
ℹ Building websocket 1.4.1
✖ Failed to build websocket 1.4.1
! Failed to build source package 'websocket', stdout + stderr (last 10 lines):

OE> ** building package indices
OE> ** installing vignettes
OE> ** testing if installed package can be loaded
OE> Error: package or namespace load failed for ‘websocket’ in dyn.load(file, DLLpath = DLLpath, ...):
OE>  unable to load shared object '/private/var/folders/gy/xtc2dqrx62s36gj1p8l_9zxh0000gn/T/RtmpDTsEVh/pkg-lib13616ccc6e36/websocket/libs/websocket.so':
OE>   dlopen(/private/var/folders/gy/xtc2dqrx62s36gj1p8l_9zxh0000gn/T/RtmpDTsEVh/pkg-lib13616ccc6e36/websocket/libs/websocket.so, 0x0006): symbol not found in flat namespace '_BIO_ctrl'
OE> Error: loading failed
OE> Execution halted
OE> ERROR: loading failed
OE> * removing ‘/private/var/folders/gy/xtc2dqrx62s36gj1p8l_9zxh0000gn/T/RtmpDTsEVh/pkg-lib13616ccc6e36/websocket’
Error in "stop_task_build(state, worker)" :
  ! Failed to build source package 'websocket'
In addition: There were 50 or more warnings (use warnings() to see the first 50)
```

Reply:
Yes I can
```r
> renv::install("websocket")
# Downloading packages -------------------------------------------------------
- Downloading websocket from CRAN ...           OK [5.3 Mb in 2.2s]
Successfully downloaded 1 package in 2.8 seconds.

The following package(s) will be installed:
- websocket [1.4.1]
These packages will be installed into "/Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library".

Do you want to proceed? [Y/n]:
# Installing packages --------------------------------------------------------
- Installing websocket ...                      OK [installed binary and cached in 2.2s]
Successfully installed 1 package in 2.4 seconds.
```